src/js/routes.js

```diff
@@ -35,6 +36,7 @@ export default (
              component={ContactUsPage}
              onEnter={redirectIfFlagIsDisabled.bind(null, flagChecks.contactUsIsEnabled())} />
       <Route path="/not-found" component={NotFound} />
+      <Route path="/blog" component={Blog} />
```

Comment:
The "/not-found" component should be the last component declared on the root '/' tree. Could we move the `/blog` upwards a bit?